phpstan.neon

```diff
@@ -1,5 +1,6 @@
 includes:
     - ./vendor/nunomaduro/larastan/extension.neon
+    - ./vendor/tpetry/laravel-postgresql-enhanced/phpstan-extension.neon
```

Comment:
As recommended in the docs: https://github.com/tpetry/laravel-postgresql-enhanced#phpstan